### DISCUSSION TITLE
ایا امکان نصب در doprax.com وجود دارد

### DISCUSSION THREAD

**Discussion by saeidghodrati** [hiddify/Hiddify-Manager#487] [General سایر]
سلام میشه روی سایت doprax.com نصب بشه کسی تستش کرده . به جای خرید سرور مجازی از قابلیت سایت doprax.com استفاده کینم لطفا راهنمایی کنید.

**Comment by zZedix** [comment]
سلام ! خیر این امکان وجود نداره !
